src/components/BuildInfo.tsx

```diff
@@ -31,3 +30,19 @@ export const BuildInfo = ({ open, onClose }: { open: boolean; onClose: () => voi
     </Dialog>
   );
 };
+
+type Build = {
+  id: string,
+  commit: string,
+  url: string
+}
+
+export function getBuild(): Build {
```

Comment:
I like the idea of bubbling this up to a lib/client class instead of reaching across components/widgets, but I also agree that we don't need the build info as its own menu item if it's in the about page. Just move this code to about/page.tsx?

Reply:
It was actually complaining that it was being called from outside of a component (guessing there is something contextual about the hook) so I just put it inside one of the about page components.